.travis/script.sh

```diff
@@ -108,14 +108,24 @@ python --version || die
 }
 
 build_cc1352() {
+    CMAKE_FLAGS="                        \
```

Comment:
We are no longer using Travis to perform build checks with arm-gcc. Can you add the relevant CMake build checks to [script/check-arm-build-cmake](https://github.com/openthread/openthread/blob/master/script/check-arm-build-cmake)?

Reply:
added cc1352 cmake check in script/check-arm-build-cmake

Reply:
Thanks, can we remove this change to `.travis/script.sh` as well?